content/docs/concepts/network/index.md

```diff
@@ -0,0 +1,437 @@
+  - name: v1
+    labels:
+      version: v1
+  - name: v2
+    labels:
+      version: v2
+    trafficPolicy:
+      loadBalancer:
+        simple: ROUND_ROBIN
+  - name: v3
+    labels:
+      version: v3
+{{< /text >}}
+
+The following diagram shows how the different configurations in the
+`my-destination-rule` destination rule affect the traffic to and from the
+`my-vtl-svc` virtual service:
+
+{{< image width="50%"
+    link="./net-config-7.svg"
+    caption="Configurable route examples defined in the destination rule"
+    >}}
+
+Visit our [destination rules reference documentation](/docs/reference/config/istio.networking.v1alpha3/#DestinationRule) to review all the enabled keys and values.
+
+## Service Entries
+
+Istio uses service entries to add services to Istio's internal service
+registry. Most commonly, you use a service entry to configure the traffic
+routes for external dependencies of the mesh. For example, you can configure
+traffic routes for APIs consumed from the web or traffic to services in legacy
+infrastructure.
+
+Adding a service to the internal registry is required to configure traffic
+routes to external services. You can enhance the configuration options of
+service entries with virtual services and destination rules. Service entries
+are not limited to external service configuration. A service entry can have one
+of two types: mesh-internal or mesh-external:
+
+-  **Mesh-internal service entries** explicitly add internal services to the
+   Istio mesh. You can use mesh-internal service entries to add services as
+   your service mesh expands to include unmanaged infrastructure. The unmanaged
+   infrastructure can include components such as VMs added to a
+   Kubernetes-based service mesh.
+
+-  **Mesh-external service entries** explicitly add external services to the
+   mesh. Mutual TLS authentication is disabled for mesh-external service
+   entries. Istio performs policy enforcement on the client-side, instead of on
+   the usual server-side used by internal service requests.
+
+Mesh-external service entries provide the following configuration options:
+
+-  You can configure multiple external dependencies with a single service
+   entry.
+-  You can configure the resolution mode for the external dependencies to
+   `NONE`, `STATIC`, or `DNS`.
+-  You can access secure external services over plain text ports to directly
+   access external dependencies from your application.
+
+The following example configuration shows a mesh-external service entry for the
+`ext-resource` external dependency:
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1alpha3
+kind: ServiceEntry
+metadata:
+  name: svc-entry
+spec:
+  hosts:
+  - ext-resource.com
+  ports:
+  - number: 443
+    name: https
+    protocol: HTTPS
+  location: MESH_EXTERNAL
+{{< /text >}}
+
+Because a service entry configuration simply adds the external resource to the
+internal service registry, you must use either a virtual service or destination
```

Comment:
Depending on the situation, you don't need anything more that a service entry to call an external service

Reply:
Great to know. @rcaballeromx can you capture that?

Reply:
Absolutely, done!